scripts/dependency_scan.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+#####
+## Build Environment Setup
+#####
+java17_0
```

Comment:
What's this do?

Reply:
This sets up java 17 in our internal build system